hack/tool/tackle

```diff
@@ -709,6 +708,24 @@ class TackleTool:
             apiJSON(self.tackle2Url + tackle2path("assessments/%d" % assmnt2['id']), self.tackle2Token, data=assmnt2, method='PATCH', ignoreErrors=ignoreErrors)
 
     def preImportCheck(self):
+        # Compatibility checks
+        # TagCategories on Hub API
+        if apiJSON(self.tackle2Url + "/hub/tagcategories", self.tackle2Token, ignoreErrors=True) is None:
+            print("ERROR: The API doesn't know TagCategories, use older version of this tool (before mta-6.1 branch).")
+            exit(1)
+
+        # TagCategories in the data directory
+        if not os.path.exists(os.path.join(self.dataDir, 'tagcategories.json')) and os.path.exists(os.path.join(self.dataDir, 'tagtypes.json')):
+            print("ERROR: The dump comes from old version using TagTypes instead of TagCategories. Update the dump first.")
+            print("Recommended steps to update the dump: (make a backup copy of the data directory first)")
```

Comment:
The tackle CLI doesn't support _update_ of exported dump to newer Hub API versions, so adding (locally tested) commands to update e.g. dump from Tackle 1.2 to be importable to current version.